src/poc/server/poc-server/src/controllers/users.ts

```diff
@@ -12,7 +17,50 @@ export const createSession = async (req: Request, res: Response) => {
     const uuid: string = uuidv4();
     console.log(uuid);
     sessionIds.push(uuid);
-    res.status(200).json({uuid});
+    res.status(200).json({ uuid });
+}
+
+/**
+ * Route to verify a users location
+ * @name /api/users/verify-location
+ * @function
+ */
+export const verifyLocation = async (req: Request, res: Response) => {
```

Comment:
this endpoint should also emit an event to the socket which has the connection that is the same as the uuid provided and send a JWT either through the socket or in the response

Reply:
added